addons/mass_mailing/security/ir.model.access.csv

```diff
@@ -7,8 +7,6 @@ access_utm_stage,utm.stage,utm.model_utm_stage,mass_mailing.group_mass_mailing_u
 access_utm_campaign_mass_mailing_user,utm.campaign,utm.model_utm_campaign,mass_mailing.group_mass_mailing_user,1,1,1,1
 access_mailing_mailing_mm_user,access.mailing.mailing.mm.user,model_mailing_mailing,mass_mailing.group_mass_mailing_user,1,1,1,1
 access_mailing_mailing_system,access.mailing.mailing.system,model_mailing_mailing,base.group_system,1,1,1,1
-access_mailing_trace_user,mailing.trace.user,model_mailing_trace,base.group_user,1,1,1,1

```

Comment:
Before this change, mailing.trace could be deleted by a regular user.

It will not be the case anymore with mail.notification;
`access_mail_notification_user,mail.notification.user,model_mail_notification,base.group_user,1,1,1,0`

To check with @tde-banana-odoo